.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-                {
-                  "name": "isAssignedToSomeone",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isLabeled",
-                  "parameters": {}
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add \"needs triage\" label to empty template",
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "needs triage :mag:"
-            }
-          },
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "Hello ${issueAuthor}, thank you for opening an issue with us! \n\nI have automatically added a \"needs triage\" label to help get things started. Our team will analyze and investigate the issue, and escalate it to the relevant team if possible. Other community members may also look into the issue and provide feedback 🙌"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "2oQJkyLzq",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "label": "for-review :book:",
-                    "projectName": "Bugs 7.1",
-                    "columnName": "In progress"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Technical 7.1",
-                    "columnName": "In progress"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Features 7.1",
-                    "columnName": "In progress"
-                  }
-                },
-                {
-                  "name": "addedToProjectColumn",
-                  "parameters": {
-                    "projectName": "Samples 7.1",
-                    "columnName": "In progress"
-                  }
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add \"In progress\" label when moved to In progress column in the project",
```

Comment:
Project board automation is not supported in the new service, so related tasks were not ported.